nextflow.config

```diff
@@ -155,8 +155,11 @@ params {
     krona_taxonomy_directory   = null
 
     // profile standardisation
-    run_profile_standardisation = false
-    generate_biom_output        = false
+    run_profile_standardisation             = false
+    taxpasta_add_taxonomy                   = false
+    taxpasta_add_samplesheet                = false
```

Comment:
I don't think this  is necessary as a user option - they don't know the exact names of the report/profile files so a user wouldn't be able to supply this themselves.

Reply:
I will remove it.

Reply:
I adressed this in the PR.